www/src/components/plugin-searchbar-body.js

```diff
@@ -195,6 +195,7 @@ class Search extends Component {
           <SearchBox translations={{ placeholder: `Search Gatsby Library` }} />
 
           <div css={{ display: `none` }}>
+            <Configure analyticsTags={['gatsby-plugins']} />
```

Comment:
`Configure` is not defined anywhere so we probably are missing some import for it?

Reply:
ah good call, I guess I was sleeping when I made the PR